Magento_Catalog/templates/product/image_with_borders.phtml

```diff
@@ -2,7 +2,8 @@
 
 <img
     class="image lazyload product-image-photo"
-    src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
-    data-src="<?= /* @escapeNotVerified */ $block->getImageUrl() ?>"
```

Comment:
why did you remove these attrs?

Reply:
Please check this section https://github.com/aFarkas/lazysizes#how-to 
Would be great to leave `data-src`